src/EPR.Calculator.Frontend.UnitTests/DefaultParameterControllerTests.cs

```diff
@@ -37,12 +39,12 @@ public async Task DefaultParamerController_Success_View_Test()
                 .Setup(_ => _.CreateClient(It.IsAny<string>()))
                 .Returns(httpClient);
 
-            var controller = new DefaultParametersController(GetConfigurationValues(), mockHttpClientFactory.Object);
+            var controller = new DefaultParametersController(ConfigurationItems.GetConfigurationValues(), mockHttpClientFactory.Object);
 
             var result = await controller.Index() as ViewResult;
             Assert.IsNotNull(result);
 
-            Assert.AreEqual(result.ViewData.Count, 11);
+            Assert.AreEqual(11, result.ViewData.Count);
```

Comment:
Instead of using 11 magic number
Can we create a readonly const and give it an appropriate name? Thanks

Reply:
Done